lib/init/routes.js

```diff
@@ -1,11 +1,38 @@
 'use strict';
 
 const _ = require('lodash');
+const config = require('config');
+const multer = require('multer');
+const mkdirp = require('mkdirp');
+
 const content = require('../content/middleware');
 const users = require('../users/middleware');
 
 module.exports = (app) => {
   return new Promise(res => {
+    // temp file storage location
+    const dest = config.storage.temp.dest || 'public/tmp/';
+
+    // multer form handling setup
+    const upload = multer({
+      storage: multer.diskStorage({
+        destination: (req, file, cb) => {
+          cb(null, dest);
+        },
+        filename: (req, file, cb) => {
+          cb(null, `/${file.originalname}`);
+        },
+      }),
+    });
+
+    // creates temp storage location if it does not exist
+    mkdirp.sync(dest);
```

Comment:
fs-extra is [not currently a dependency](https://github.com/punchcard-cms/punchcard/blob/master/package.json#L59), which is why I chose this more targeted module. Can switch if that's preferred.